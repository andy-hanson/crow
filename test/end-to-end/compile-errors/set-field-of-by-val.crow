main void()
	r mut r = 0,
	r.x := 3

r record by-val, mut
	x mut nat
