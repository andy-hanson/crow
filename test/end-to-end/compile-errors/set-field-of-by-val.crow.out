exit code: 1
stderr:
test/end-to-end/compile-errors/set-field-of-by-val.crow 3:9-3:11 There is no function named 'set-x'.
Uncaught exception: Error: Reached compile error
	at main
